### PR TITLE
fullstack templates use @wix/wix-test-env

### PR DESCRIPTION
**Pull request by hugebdu**
~@ranyitz draft~
~WDYT?~

@ranyitz :dropthemic:

### CODE REVIEW

File: packages/create-yoshi-app/templates/fullstack/typescript/index-dev.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { env } from './environment';
+
+(async () => {
+  await env.start();
+})();
```
Comment (ranyitz) [inline packages/create-yoshi-app/templates/fullstack/typescript/index-dev.ts]:
```suggestion
env.start();
```

I think that the `async` IIFE + `await` became redundant

File: packages/create-yoshi-app/templates/fullstack/javascript/index-dev.js [hunk 2]
```diff
@@ -1,8 +1,5 @@
-const { bootstrapServer, emitConfigs } = require('./environment');
+const { env } = require('./environment');
 
 (async () => {
-  const app = bootstrapServer();
-
-  await emitConfigs();
-  await app.start();
+  await env.start();
```
Comment (ranyitz) [inline packages/create-yoshi-app/templates/fullstack/javascript/index-dev.js]: @hugebdu Would be nice to change this as well

File: packages/create-yoshi-app/templates/fullstack/typescript/__tests__/server.e2e.ts [hunk 3]
```diff
@@ -1,9 +1,6 @@
-import axios from 'axios';
-
 describe('When rendering', () => {
   it('should display a title', async () => {
-    const url = app.getUrl('/');
-    const response = await axios.get(url);
+    const response = await axios.get('/');
```
Comment (yanivefraim) [inline packages/create-yoshi-app/templates/fullstack/typescript/__tests__/server.e2e.ts]: we should change this in all other server e2e tests (fullstack-js, server template etc.)

**Comment by saarkuriel** [comment]
I really like it too and I agree with @yanivefraim.

**Comment by ranyitz** [comment]
I'll summaries the comments:

* [ ] [javascript server it test axios change](https://github.com/hugebdu/yoshi/blob/256af8d22f1b6ad3c92cd17c6eeb744ffd9897b2/packages/create-yoshi-app/templates/server/javascript/test/it/server.it.js#L9-L10)

* [ ] [typescript server it test axios change](https://github.com/hugebdu/yoshi/blob/256af8d22f1b6ad3c92cd17c6eeb744ffd9897b2/packages/create-yoshi-app/templates/server/typescript/test/it/server.it.ts#L9-L10)

* [ ] [javascript fullstack server test axios change](https://github.com/hugebdu/yoshi/blob/256af8d22f1b6ad3c92cd17c6eeb744ffd9897b2/packages/create-yoshi-app/templates/fullstack/javascript/__tests__/server.e2e.js#L5-L6)

Regarding fed-handbook modification @yanivefraim @saarkuriel how much work should be done? One of you will have the time to do this before the crash course starts?

File: packages/create-yoshi-app/templates/fullstack/typescript/__tests__/server.e2e.ts (hunk 3 above)
Comment (hugebdu) [reply packages/create-yoshi-app/templates/fullstack/typescript/__tests__/server.e2e.ts]: @yanivefraim [updated](https://github.com/wix/yoshi/pull/2021/commits/1682b9ef15c29365177dc7cdc8ac61792bc2fc6b)

**Pull request merged**